gateway/src/apicast/policy/scope_check/apicast-policy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "Scope Check",
+  "summary": "Adds scope check.",
+  "description": [
+    "This policy adds scope check.\n",
+    "This policy exposes realm roles and client roles in the access token."
+  ],
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "scopes": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "roles": {
+              "type": "array",
+              "description": "Roles necessary to access the resource",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "role": {
+                    "type": "string",
+                    "description": "Name of the role"
```

Comment:
I'm not really sure who's is this role. User's ?

Then we should name it clearly. Right now "role" vs "client" does not really say much.

Reply:
Clarified the realm roles and the client roles.